modules/gitops-argo-cd-installation.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * argo-cd-custom-resource-properties.adoc
+
+:_content-type: PROCEDURE
+[id="gitops-argo-cd-installation_{context}"]
+= Installing Argo CD 
+
+[role="_abstract"]
+To manage cluster configurations or deploy applications, you can install and deploy a new Argo CD instance. To install the Argo CD instance follow the steps given below:
+
+.Procedure
```

Comment:
Please update this procedure so that it refers to the web console and describes user interactions with the interface in a way that is more consistent with other similar topics. For examples, see `modules/cluster-logging-deploy-console.adoc` or search the openshift-docs repo for `*Operators* ->`.